.eslintrc.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
   extends: '@amnis/eslint-config-react',
+  rules: {
+    'linebreak-style': ['error', 'windows'],
+  },
```

Comment:
Need this new linting rule removed.

The repository uses unix line-endings. If you opened the cloned repository with VSCode, this setting should have been set to default. In VS Code, the line-ending sequence is shown on the bottom bar.

![image](https://user-images.githubusercontent.com/41806292/165994116-512e5a02-5961-4ac5-8310-2c4d9d8f4f58.png)

You should be able to fix **all** the line endings easily with `yarn lint --fix`.

Reply:
We can delete this rule. This should already come with the `@amnis/eslint-config-react` configuration.